src/serlo-editor-repo/plugin-text/components/text-editor.tsx

```diff
@@ -270,6 +276,13 @@ export function TextEditor(props: TextEditorProps) {
     if (files?.length > 0) {
       const imagePluginState = plugins.image?.onFiles?.(files)
       if (imagePluginState !== undefined) {
+        if (isListActive) {
+          if (!loggedInData) return
+          const { noImagePasteInLists } = loggedInData.strings.editor.image
+          showToastNotice(noImagePasteInLists, 'warning', 6000)
```

Comment:
I think the default duration (3.2s) would be enough here.

Reply:
Thanks for the hint, updated :)